### PR TITLE
CDPT-1935: Add  Survey Notification Banner

### PR DESCRIPTION
**Pull request by MeenaModhvadia**
## Description
The Survey Notification Banner has been added above 'Other ways to prepare using'  if present on page otherwise added above Other ways to

## Self-review checklist

* [ ] (1) Quick stakeholder demo done OR
* [ ] (2) ...bug with before and after screenshots
* [x] (3) Tests passing
* [ ] (4) Branch ready to be merged (not work in progress)
* [ ] (5) No superfluous changes in diff
* [x] (6) No TODO's without new ticket numbers
* [x] (7) PR Prefixed with ticket number e.g. `CT-7654 ...`

### Screenshots

image
image

### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CDPT-1935

### Deployment
None

### Manual testing instructions
None

### CODE REVIEW

File: app/views/pages/arbitration.html.erb [hunk 1]
```diff
@@ -275,6 +275,9 @@
   </div>
 </section>
 
+<br>
```
Comment (vertism) [inline app/views/pages/arbitration.html.erb:278]: Rather than adding br tags to every page, can you use css to create the necessary space before the survey banner?

File: app/views/shared/_prepare_survey_link.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<%= render "shared/survey_link" %>
```
Comment (vertism) [inline app/views/shared/_prepare_survey_link.html.erb:1]: Is this file needed?

File: app/views/shared/_survey_link.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+  <div class="govuk-notification-banner__header">
+    <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+      Important
```
Comment (vertism) [inline app/views/shared/_survey_link.html.erb:4]: Should the title be "Help us improve our website"?

File: app/views/pages/arbitration.html.erb (hunk 1 above)
Comment (vertism) [reply app/views/pages/arbitration.html.erb:278]: Maybe wrap it with grid divs like app/views/shared/_links.html.erb

File: app/views/shared/_survey_link.html.erb (hunk 3 above)
Comment (MeenaModhvadia) [reply app/views/shared/_survey_link.html.erb]: Renamed iid="govuk-notification-banner-title" and aria-labelledby="govuk-notification-banner-title"

File: app/views/shared/_prepare_survey_link.html.erb (hunk 2 above)
Comment (MeenaModhvadia) [reply app/views/shared/_prepare_survey_link.html.erb]: Deleted

File: app/views/shared/_survey_link.html.erb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<div class="govuk-grid-row">
+  <div class="govuk-notification-banner" role="region" aria-labelledby="help_us_improve_our_website" data-module="govuk-notification-banner">
```
Comment (vertism) [inline app/views/shared/_survey_link.html.erb:2]: This can all be done using the helper from govuk-components - https://govuk-components.netlify.app/components/notification-banner/#input-erb-notification-banner-with-a-block.

Comment (vertism) [reply app/views/shared/_survey_link.html.erb:2]: Also wanted to check again that the title should be "Important" as the ticket doesn't specify that.

Comment (MeenaModhvadia) [reply app/views/shared/_survey_link.html.erb:2]: I have changed the title to Help us improve our website instead of Important.

I'm not clear what can be done using the helper from govuk-components - https://govuk-components.netlify.app/components/notification-banner/#input-erb-notification-banner-with-a-block.

Do you mean the line 1?

Comment (vertism) [reply app/views/shared/_survey_link.html.erb:2]: You can replace all the banner code with something like the example on that page:
```
<%= govuk_notification_banner(title_text: "Important") do %>
  <p>
    You have 7 days left to send your application. You can send it:
  </p>
  <ul>
    <li>by email</li>
    <li>by post</li>
    <li>by courier</li>
  </ul>
<% end %>
```

File: app/views/shared/_survey_link.html.erb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<%= govuk_notification_banner(title_text: "Help us improve our website") do %>
+  <ul>
```
Comment (vertism) [inline app/views/shared/_survey_link.html.erb:2]: Why is it in a list?

File: spec/requests/static_pages_spec.rb [hunk 6]
```diff
@@ -3,66 +3,85 @@
...
   it "shows the professional mediation page" do
     get professional_mediation_path
     expect(response).to be_successful
     expect(response.body).to include("Mediation sessions are run by professionals who help you try to reach an agreement")
+    expect(response.body).to include("Help us improve our website")
   end
 
   it "shows the professional mediation other parent page" do
     get professional_mediation_other_parent_path
+    puts professional_mediation_other_parent_path
```
Comment (vertism) [inline spec/requests/static_pages_spec.rb:74]: Puts left in

**Review by vertism — APPROVED** [review]
LGTM
